src/system/SystemLayer.h

```diff
@@ -54,34 +54,17 @@ namespace System {
 
 class Layer;
 class Timer;
-
-#if CHIP_SYSTEM_CONFIG_USE_LWIP
 class Object;
-#endif // CHIP_SYSTEM_CONFIG_USE_LWIP
 
 namespace Platform {
-namespace Layer {
+namespace EventSupport {
```

Comment:
nitpick: do we need "Support" in the name here? I generally claim that support/helper/core/util and similar naming do not add significant readability. Could we get around with just "Eventing" or "Events" ?

Reply:
Sure, will rename in the next PR.